{"event": "config", "data": {"system": {"arch": "cuda", "sshkey": null, "nodes": [{"name": "local", "ip": "127.0.0.1", "sshport": 22, "user": "root", "main": true, "hostname": "cn-e002.server.mila.quebec", "local": true}], "self": {"name": "local", "ip": "127.0.0.1", "sshport": 22, "user": "root", "main": true, "hostname": "cn-e002.server.mila.quebec", "local": true}}, "dirs": {"base": "/network/scratch/o/ortizgas/data/milabench", "venv": "/network/scratch/o/ortizgas/data/milabench/venv/torch", "data": "/network/scratch/o/ortizgas/data/milabench/data", "runs": "/network/scratch/o/ortizgas/data/milabench/runs", "extra": "/network/scratch/o/ortizgas/data/milabench/extra/diffusion-gpus", "cache": "/network/scratch/o/ortizgas/data/milabench/cache"}, "group": "diffusion-gpus", "install_group": "torch", "install_variant": "cuda", "run_name": "Tesla-V100-SXM2-32GB-LS_diffusion-gpus_6.staging", "enabled": true, "capabilities": {"nodes": 1}, "max_duration": 600, "voir": {"options": {"stop": 60, "interval": "1s"}}, "validation": {"usage": {"gpu_load_threshold": 0.5, "gpu_mem_threshold": 0.5}}, "config_base": "/home/mila/o/ortizgas/CODE/milabench/config", "config_file": "/home/mila/o/ortizgas/CODE/milabench/config/standard.yaml", "definition": "/home/mila/o/ortizgas/CODE/milabench/benchmarks/diffusion", "tags": ["diffusion", "multigpu"], "plan": {"method": "njobs", "n": 1}, "argv": {"--num_epochs": 5, "--batch_size": 32, "--num_workers": "auto({n_worker}, 8)", "--cache": "{milabench_cache}"}, "num_machines": 1, "weight": 1.0, "name": "diffusion-gpus", "tag": ["diffusion-gpus", "0"], "job-number": 0, "devices": [0, 1, 2, 3]}, "pipe": null}
{"event": "meta", "data": {"cpu": {"count": 40, "brand": "Intel(R) Xeon(R) CPU E5-2698 v4 @ 2.20GHz"}, "os": {"sysname": "Linux", "nodename": "cn-e002.server.mila.quebec", "release": "5.15.0-1048-nvidia", "version": "#48-Ubuntu SMP Thu Mar 21 18:19:02 UTC 2024", "machine": "x86_64"}, "accelerators": {"arch": "cuda", "gpus": {"0": {"minor_number": "0", "device": 0, "product": "Tesla V100-SXM2-32GB-LS", "memory": {"used": 267.0, "total": 32768.0}, "utilization": {"compute": 0.0, "memory": 0}, "temperature": 37.0, "power": 45.451, "selection_variable": "CUDA_VISIBLE_DEVICES"}, "1": {"minor_number": "1", "device": 1, "product": "Tesla V100-SXM2-32GB-LS", "memory": {"used": 267.0, "total": 32768.0}, "utilization": {"compute": 0.0, "memory": 0}, "temperature": 38.0, "power": 45.451, "selection_variable": "CUDA_VISIBLE_DEVICES"}, "2": {"minor_number": "5", "device": 2, "product": "Tesla V100-SXM2-32GB-LS", "memory": {"used": 267.0, "total": 32768.0}, "utilization": {"compute": 0.0, "memory": 0}, "temperature": 34.0, "power": 42.103, "selection_variable": "CUDA_VISIBLE_DEVICES"}, "3": {"minor_number": "6", "device": 3, "product": "Tesla V100-SXM2-32GB-LS", "memory": {"used": 267.0, "total": 32768.0}, "utilization": {"compute": 0.0, "memory": 0}, "temperature": 36.0, "power": 43.58, "selection_variable": "CUDA_VISIBLE_DEVICES"}}}, "date": 1734054119.590146, "milabench": {"tag": "v1.0.0_RC1-13-g7a90b16", "commit": "7a90b1691650232ecd63abded0b7be84bb294c05", "date": "2024-11-08 11:28:56 -0500"}, "pytorch": {"torch": "2.4.0+cu121", "compiler": "GCC 9.3", "cpp": "C++ Version: 201703", "intel": "Intel(R) oneAPI Math Kernel Library Version 2022.2-Product Build 20220804 for Intel(R) 64 architecture applications", "mkl": "OpenMP 201511 (a.k.a. OpenMP 4.5)", "openmp": "OpenMP 201511 (a.k.a. OpenMP 4.5)", "lapack": "LAPACK is enabled (usually provided by MKL)", "nnpack": "NNPACK is enabled", "cpu": "CPU capability usage: AVX2", "build_settings": {"BLAS_INFO": "mkl", "BUILD_TYPE": "Release", "CUDA_VERSION": "12.1", "CUDNN_VERSION": "9.1.0", "CXX_COMPILER": "/opt/rh/devtoolset-9/root/usr/bin/c++", "CXX_FLAGS": "-D_GLIBCXX_USE_CXX11_ABI=0 -fabi-version=11 -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -DNDEBUG -DUSE_KINETO -DLIBKINETO_NOROCTRACER -DUSE_FBGEMM -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DSYMBOLICATE_MOBILE_DEBUG_HANDLE -O2 -fPIC -Wall -Wextra -Werror=return-type -Werror=non-virtual-dtor -Werror=bool-operation -Wnarrowing -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-function -Wno-unused-result -Wno-strict-overflow -Wno-strict-aliasing -Wno-stringop-overflow -Wsuggest-override -Wno-psabi -Wno-error=pedantic -Wno-error=old-style-cast -Wno-missing-braces -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow", "LAPACK_INFO": "mkl", "PERF_WITH_AVX": "1", "PERF_WITH_AVX2": "1", "PERF_WITH_AVX512": "1", "TORCH_VERSION": "2.4.0", "USE_CUDA": "ON", "USE_CUDNN": "ON", "USE_CUSPARSELT": "1", "USE_EXCEPTION_PTR": "1", "USE_GFLAGS": "OFF", "USE_GLOG": "OFF", "USE_GLOO": "ON", "USE_MKL": "ON", "USE_MKLDNN": "ON", "USE_MPI": "OFF", "USE_NCCL": "1", "USE_NNPACK": "ON", "USE_OPENMP": "ON", "USE_ROCM": "OFF", "USE_ROCM_KERNEL_ASSERT": "OFF"}}}, "pipe": null}
{"event": "start", "data": {"command": ["/home/mila/o/ortizgas/CODE/milabench/milabench/scripts/activator", "/network/scratch/o/ortizgas/data/milabench/venv/torch", "/network/scratch/o/ortizgas/data/milabench/cache", "accelerate", "launch", "--mixed_precision=bf16", "--dynamo_backend=no", "--machine_rank=0", "--num_machines=1", "--multi_gpu", "--gradient_accumulation_steps=1", "--num_cpu_threads_per_process=4", "--main_process_ip=127.0.0.1", "--main_process_port=29400", "--num_processes=4", "/home/mila/o/ortizgas/CODE/milabench/benchmarks/diffusion/main.py", "--num_epochs", "5", "--batch_size", "6", "--num_workers", "8", "--cache", "/network/scratch/o/ortizgas/data/milabench/cache"], "time": 1734036119.6749237}, "pipe": null}
{"event": "line", "data": "/network/scratch/o/ortizgas/data/milabench/cache\n", "pipe": "stdout"}
{"event": "line", "data": "/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/xformers/ops/fmha/flash.py:211: FutureWarning: `torch.library.impl_abstract` was renamed to `torch.library.register_fake`. Please use that instead; we will remove `torch.library.impl_abstract` in a future version of PyTorch.\n", "pipe": "stderr"}
{"event": "line", "data": "  @torch.library.impl_abstract(\"xformers_flash::flash_fwd\")\n", "pipe": "stderr"}
{"event": "line", "data": "/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/xformers/ops/fmha/flash.py:211: FutureWarning: `torch.library.impl_abstract` was renamed to `torch.library.register_fake`. Please use that instead; we will remove `torch.library.impl_abstract` in a future version of PyTorch.\n", "pipe": "stderr"}
{"event": "line", "data": "  @torch.library.impl_abstract(\"xformers_flash::flash_fwd\")\n", "pipe": "stderr"}
{"event": "line", "data": "/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/xformers/ops/fmha/flash.py:211: FutureWarning: `torch.library.impl_abstract` was renamed to `torch.library.register_fake`. Please use that instead; we will remove `torch.library.impl_abstract` in a future version of PyTorch.\n", "pipe": "stderr"}
{"event": "line", "data": "  @torch.library.impl_abstract(\"xformers_flash::flash_fwd\")\n", "pipe": "stderr"}
{"event": "line", "data": "/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/xformers/ops/fmha/flash.py:211: FutureWarning: `torch.library.impl_abstract` was renamed to `torch.library.register_fake`. Please use that instead; we will remove `torch.library.impl_abstract` in a future version of PyTorch.\n", "pipe": "stderr"}
{"event": "line", "data": "  @torch.library.impl_abstract(\"xformers_flash::flash_fwd\")\n", "pipe": "stderr"}
{"event": "line", "data": "/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/xformers/ops/fmha/flash.py:344: FutureWarning: `torch.library.impl_abstract` was renamed to `torch.library.register_fake`. Please use that instead; we will remove `torch.library.impl_abstract` in a future version of PyTorch.\n", "pipe": "stderr"}
{"event": "line", "data": "  @torch.library.impl_abstract(\"xformers_flash::flash_bwd\")\n", "pipe": "stderr"}
{"event": "line", "data": "/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/xformers/ops/fmha/flash.py:344: FutureWarning: `torch.library.impl_abstract` was renamed to `torch.library.register_fake`. Please use that instead; we will remove `torch.library.impl_abstract` in a future version of PyTorch.\n", "pipe": "stderr"}
{"event": "line", "data": "  @torch.library.impl_abstract(\"xformers_flash::flash_bwd\")\n", "pipe": "stderr"}
{"event": "line", "data": "/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/xformers/ops/fmha/flash.py:344: FutureWarning: `torch.library.impl_abstract` was renamed to `torch.library.register_fake`. Please use that instead; we will remove `torch.library.impl_abstract` in a future version of PyTorch.\n", "pipe": "stderr"}
{"event": "line", "data": "  @torch.library.impl_abstract(\"xformers_flash::flash_bwd\")\n", "pipe": "stderr"}
{"event": "line", "data": "/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/xformers/ops/fmha/flash.py:344: FutureWarning: `torch.library.impl_abstract` was renamed to `torch.library.register_fake`. Please use that instead; we will remove `torch.library.impl_abstract` in a future version of PyTorch.\n", "pipe": "stderr"}
{"event": "line", "data": "  @torch.library.impl_abstract(\"xformers_flash::flash_bwd\")\n", "pipe": "stderr"}
{"event": "line", "data": "No overseer found\n", "pipe": "stdout"}
{"event": "line", "data": "No overseer found\n", "pipe": "stdout"}
{"event": "line", "data": "No overseer found\n", "pipe": "stdout"}
{"event": "line", "data": "No overseer found\n", "pipe": "stdout"}
{"event": "line", "data": "Repo card metadata block was not found. Setting CardData to empty.\n", "pipe": "stderr"}
{"event": "line", "data": "Repo card metadata block was not found. Setting CardData to empty.\n", "pipe": "stderr"}
{"event": "line", "data": "Repo card metadata block was not found. Setting CardData to empty.\n", "pipe": "stderr"}
{"event": "line", "data": "Repo card metadata block was not found. Setting CardData to empty.\n", "pipe": "stderr"}
{"event": "line", "data": "/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/transformers/tokenization_utils_base.py:1601: FutureWarning: `clean_up_tokenization_spaces` was not set. It will be set to `True` by default. This behavior will be depracted in transformers v4.45, and will be then set to `False` by default. For more details check this issue: https://github.com/huggingface/transformers/issues/31884\n", "pipe": "stderr"}
{"event": "line", "data": "  warnings.warn(\n", "pipe": "stderr"}
{"event": "line", "data": "/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/transformers/tokenization_utils_base.py:1601: FutureWarning: `clean_up_tokenization_spaces` was not set. It will be set to `True` by default. This behavior will be depracted in transformers v4.45, and will be then set to `False` by default. For more details check this issue: https://github.com/huggingface/transformers/issues/31884\n", "pipe": "stderr"}
{"event": "line", "data": "  warnings.warn(\n", "pipe": "stderr"}
{"event": "line", "data": "/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/transformers/tokenization_utils_base.py:1601: FutureWarning: `clean_up_tokenization_spaces` was not set. It will be set to `True` by default. This behavior will be depracted in transformers v4.45, and will be then set to `False` by default. For more details check this issue: https://github.com/huggingface/transformers/issues/31884\n", "pipe": "stderr"}
{"event": "line", "data": "  warnings.warn(\n", "pipe": "stderr"}
{"event": "line", "data": "/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/transformers/tokenization_utils_base.py:1601: FutureWarning: `clean_up_tokenization_spaces` was not set. It will be set to `True` by default. This behavior will be depracted in transformers v4.45, and will be then set to `False` by default. For more details check this issue: https://github.com/huggingface/transformers/issues/31884\n", "pipe": "stderr"}
{"event": "line", "data": "  warnings.warn(\n", "pipe": "stderr"}
{"event": "data", "data": {"task": "main", "time": 1734036162.8962622, "gpudata": {"0": {"memory": [4902.625, 32768.0], "load": 0.15, "temperature": 35.0, "power": 50.756}, "1": {"memory": [4998.625, 32768.0], "load": 0.02, "temperature": 36.0, "power": 50.24}, "2": {"memory": [4998.625, 32768.0], "load": 0.0, "temperature": 32.0, "power": 47.38}, "3": {"memory": [4902.625, 32768.0], "load": 0.01, "temperature": 35.0, "power": 48.859}}}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "time": 1734036162.8962622, "iodata": {"read_count": 0, "write_count": 11, "read_bytes": 0, "read_time": 0, "write_time": 2, "busy_time": 12}}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "time": 1734036162.8962622, "netdata": {"bytes_sent": 24669526267216, "bytes_recv": 25088580889445, "packets_sent": 2798456860, "packets_recv": 3100787198, "errin": 0, "errout": 0, "dropin": 0, "dropout": 0}}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "time": 1734036162.8962622, "cpudata": {"memory": [69581340672, 540098572288], "load": 0.0}}, "pipe": "data"}
{"event": "data", "data": {"progress": [0, 60], "task": "early_stop"}, "pipe": "data"}
{"event": "data", "data": {"progress": [1, 60], "task": "early_stop"}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "time": 1734036172.913207, "gpudata": {"0": {"memory": [32374.625, 32768.0], "load": 1.0, "temperature": 40.0, "power": 186.945}, "1": {"memory": [32470.625, 32768.0], "load": 1.0, "temperature": 41.0, "power": 174.969}, "2": {"memory": [32470.625, 32768.0], "load": 1.0, "temperature": 38.0, "power": 155.16}, "3": {"memory": [32374.625, 32768.0], "load": 1.0, "temperature": 40.0, "power": 148.059}}}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "time": 1734036172.913207, "iodata": {"read_count": 0, "write_count": 78, "read_bytes": 0, "read_time": 0, "write_time": 552, "busy_time": 572}}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "time": 1734036172.913207, "netdata": {"bytes_sent": 24669527563504, "bytes_recv": 25088582128110, "packets_sent": 2798458232, "packets_recv": 3100788556, "errin": 0, "errout": 0, "dropin": 0, "dropout": 0}}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "time": 1734036172.913207, "cpudata": {"memory": [75585925120, 540098572288], "load": 24.6}}, "pipe": "data"}
{"event": "data", "data": {"progress": [2, 60], "task": "early_stop"}, "pipe": "data"}
{"event": "line", "data": "[rank2]: Traceback (most recent call last):\n", "pipe": "stderr"}
{"event": "line", "data": "[rank2]:   File \"/home/mila/o/ortizgas/CODE/milabench/benchmarks/diffusion/main.py\", line 255, in <module>\n", "pipe": "stderr"}
{"event": "line", "data": "[rank2]:     main()\n", "pipe": "stderr"}
{"event": "line", "data": "[rank2]:   File \"/home/mila/o/ortizgas/CODE/milabench/benchmarks/diffusion/main.py\", line 249, in main\n", "pipe": "stderr"}
{"event": "line", "data": "[rank2]:     train(observer, config)\n", "pipe": "stderr"}
{"event": "line", "data": "[rank2]:   File \"/home/mila/o/ortizgas/CODE/milabench/benchmarks/diffusion/main.py\", line 209, in train\n", "pipe": "stderr"}
{"event": "line", "data": "[rank2]:     accelerator.backward(loss)\n", "pipe": "stderr"}
{"event": "line", "data": "[rank2]:   File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/accelerate/accelerator.py\", line 2196, in backward\n", "pipe": "stderr"}
{"event": "line", "data": "[rank2]:     loss.backward(**kwargs)\n", "pipe": "stderr"}
{"event": "line", "data": "[rank2]:   File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/torch/_tensor.py\", line 521, in backward\n", "pipe": "stderr"}
{"event": "line", "data": "[rank2]:     torch.autograd.backward(\n", "pipe": "stderr"}
{"event": "line", "data": "[rank2]:   File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/torch/autograd/__init__.py\", line 289, in backward\n", "pipe": "stderr"}
{"event": "line", "data": "[rank2]:     _engine_run_backward(\n", "pipe": "stderr"}
{"event": "line", "data": "[rank2]:   File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/torch/autograd/graph.py\", line 768, in _engine_run_backward\n", "pipe": "stderr"}
{"event": "line", "data": "[rank2]:     return Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass\n", "pipe": "stderr"}
{"event": "line", "data": "[rank2]: torch.OutOfMemoryError: CUDA out of memory. Tried to allocate 960.00 MiB. GPU 2 has a total capacity of 31.74 GiB of which 317.38 MiB is free. Including non-PyTorch memory, this process has 31.43 GiB memory in use. Of the allocated memory 29.26 GiB is allocated by PyTorch, and 1.58 GiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)\n", "pipe": "stderr"}
{"event": "line", "data": "[rank3]: Traceback (most recent call last):\n", "pipe": "stderr"}
{"event": "line", "data": "[rank3]:   File \"/home/mila/o/ortizgas/CODE/milabench/benchmarks/diffusion/main.py\", line 255, in <module>\n", "pipe": "stderr"}
{"event": "line", "data": "[rank3]:     main()\n", "pipe": "stderr"}
{"event": "line", "data": "[rank3]:   File \"/home/mila/o/ortizgas/CODE/milabench/benchmarks/diffusion/main.py\", line 249, in main\n", "pipe": "stderr"}
{"event": "line", "data": "[rank3]:     train(observer, config)\n", "pipe": "stderr"}
{"event": "line", "data": "[rank3]:   File \"/home/mila/o/ortizgas/CODE/milabench/benchmarks/diffusion/main.py\", line 209, in train\n", "pipe": "stderr"}
{"event": "line", "data": "[rank3]:     accelerator.backward(loss)\n", "pipe": "stderr"}
{"event": "line", "data": "[rank3]:   File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/accelerate/accelerator.py\", line 2196, in backward\n", "pipe": "stderr"}
{"event": "line", "data": "[rank3]:     loss.backward(**kwargs)\n", "pipe": "stderr"}
{"event": "line", "data": "[rank3]:   File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/torch/_tensor.py\", line 521, in backward\n", "pipe": "stderr"}
{"event": "line", "data": "[rank3]:     torch.autograd.backward(\n", "pipe": "stderr"}
{"event": "line", "data": "[rank3]:   File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/torch/autograd/__init__.py\", line 289, in backward\n", "pipe": "stderr"}
{"event": "line", "data": "[rank3]:     _engine_run_backward(\n", "pipe": "stderr"}
{"event": "line", "data": "[rank3]:   File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/torch/autograd/graph.py\", line 768, in _engine_run_backward\n", "pipe": "stderr"}
{"event": "line", "data": "[rank3]:     return Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass\n", "pipe": "stderr"}
{"event": "line", "data": "[rank3]: torch.OutOfMemoryError: CUDA out of memory. Tried to allocate 960.00 MiB. GPU 3 has a total capacity of 31.74 GiB of which 413.38 MiB is free. Including non-PyTorch memory, this process has 31.33 GiB memory in use. Of the allocated memory 29.26 GiB is allocated by PyTorch, and 1.58 GiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)\n", "pipe": "stderr"}
{"event": "line", "data": "[rank1]: Traceback (most recent call last):\n", "pipe": "stderr"}
{"event": "line", "data": "[rank1]:   File \"/home/mila/o/ortizgas/CODE/milabench/benchmarks/diffusion/main.py\", line 255, in <module>\n", "pipe": "stderr"}
{"event": "line", "data": "[rank1]:     main()\n", "pipe": "stderr"}
{"event": "line", "data": "[rank1]:   File \"/home/mila/o/ortizgas/CODE/milabench/benchmarks/diffusion/main.py\", line 249, in main\n", "pipe": "stderr"}
{"event": "line", "data": "[rank1]:     train(observer, config)\n", "pipe": "stderr"}
{"event": "line", "data": "[rank1]:   File \"/home/mila/o/ortizgas/CODE/milabench/benchmarks/diffusion/main.py\", line 209, in train\n", "pipe": "stderr"}
{"event": "line", "data": "[rank1]:     accelerator.backward(loss)\n", "pipe": "stderr"}
{"event": "line", "data": "[rank1]:   File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/accelerate/accelerator.py\", line 2196, in backward\n", "pipe": "stderr"}
{"event": "line", "data": "[rank1]:     loss.backward(**kwargs)\n", "pipe": "stderr"}
{"event": "line", "data": "[rank1]:   File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/torch/_tensor.py\", line 521, in backward\n", "pipe": "stderr"}
{"event": "line", "data": "[rank1]:     torch.autograd.backward(\n", "pipe": "stderr"}
{"event": "line", "data": "[rank1]:   File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/torch/autograd/__init__.py\", line 289, in backward\n", "pipe": "stderr"}
{"event": "line", "data": "[rank1]:     _engine_run_backward(\n", "pipe": "stderr"}
{"event": "line", "data": "[rank1]:   File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/torch/autograd/graph.py\", line 768, in _engine_run_backward\n", "pipe": "stderr"}
{"event": "line", "data": "[rank1]:     return Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass\n", "pipe": "stderr"}
{"event": "line", "data": "[rank1]: torch.OutOfMemoryError: CUDA out of memory. Tried to allocate 960.00 MiB. GPU 1 has a total capacity of 31.74 GiB of which 317.38 MiB is free. Including non-PyTorch memory, this process has 31.43 GiB memory in use. Of the allocated memory 29.26 GiB is allocated by PyTorch, and 1.58 GiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)\n", "pipe": "stderr"}
{"event": "line", "data": "[rank0]: Traceback (most recent call last):\n", "pipe": "stderr"}
{"event": "line", "data": "[rank0]:   File \"/home/mila/o/ortizgas/CODE/milabench/benchmarks/diffusion/main.py\", line 255, in <module>\n", "pipe": "stderr"}
{"event": "line", "data": "[rank0]:     main()\n", "pipe": "stderr"}
{"event": "line", "data": "[rank0]:   File \"/home/mila/o/ortizgas/CODE/milabench/benchmarks/diffusion/main.py\", line 249, in main\n", "pipe": "stderr"}
{"event": "line", "data": "[rank0]:     train(observer, config)\n", "pipe": "stderr"}
{"event": "line", "data": "[rank0]:   File \"/home/mila/o/ortizgas/CODE/milabench/benchmarks/diffusion/main.py\", line 209, in train\n", "pipe": "stderr"}
{"event": "line", "data": "[rank0]:     accelerator.backward(loss)\n", "pipe": "stderr"}
{"event": "line", "data": "[rank0]:   File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/accelerate/accelerator.py\", line 2196, in backward\n", "pipe": "stderr"}
{"event": "line", "data": "[rank0]:     loss.backward(**kwargs)\n", "pipe": "stderr"}
{"event": "line", "data": "[rank0]:   File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/torch/_tensor.py\", line 521, in backward\n", "pipe": "stderr"}
{"event": "line", "data": "[rank0]:     torch.autograd.backward(\n", "pipe": "stderr"}
{"event": "line", "data": "[rank0]:   File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/torch/autograd/__init__.py\", line 289, in backward\n", "pipe": "stderr"}
{"event": "line", "data": "[rank0]:     _engine_run_backward(\n", "pipe": "stderr"}
{"event": "line", "data": "[rank0]:   File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/torch/autograd/graph.py\", line 768, in _engine_run_backward\n", "pipe": "stderr"}
{"event": "line", "data": "[rank0]:     return Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass\n", "pipe": "stderr"}
{"event": "line", "data": "[rank0]: torch.OutOfMemoryError: CUDA out of memory. Tried to allocate 960.00 MiB. GPU 0 has a total capacity of 31.74 GiB of which 413.38 MiB is free. Including non-PyTorch memory, this process has 31.33 GiB memory in use. Of the allocated memory 29.26 GiB is allocated by PyTorch, and 1.58 GiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)\n", "pipe": "stderr"}
{"event": "line", "data": "W1212 15:42:59.392334 139878894232704 torch/distributed/elastic/multiprocessing/api.py:858] Sending process 988008 closing signal SIGTERM\n", "pipe": "stderr"}
{"event": "line", "data": "W1212 15:42:59.392600 139878894232704 torch/distributed/elastic/multiprocessing/api.py:858] Sending process 988010 closing signal SIGTERM\n", "pipe": "stderr"}
{"event": "line", "data": "W1212 15:42:59.392752 139878894232704 torch/distributed/elastic/multiprocessing/api.py:858] Sending process 988011 closing signal SIGTERM\n", "pipe": "stderr"}
{"event": "line", "data": "E1212 15:42:59.971853 139878894232704 torch/distributed/elastic/multiprocessing/api.py:833] failed (exitcode: 1) local_rank: 1 (pid: 988009) of binary: /network/scratch/o/ortizgas/data/milabench/venv/torch/bin/python\n", "pipe": "stderr"}
{"event": "line", "data": "Traceback (most recent call last):\n", "pipe": "stderr"}
{"event": "line", "data": "  File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/bin/accelerate\", line 8, in <module>\n", "pipe": "stderr"}
{"event": "line", "data": "    sys.exit(main())\n", "pipe": "stderr"}
{"event": "line", "data": "  File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/accelerate/commands/accelerate_cli.py\", line 48, in main\n", "pipe": "stderr"}
{"event": "line", "data": "    args.func(args)\n", "pipe": "stderr"}
{"event": "line", "data": "  File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/accelerate/commands/launch.py\", line 1165, in launch_command\n", "pipe": "stderr"}
{"event": "line", "data": "    multi_gpu_launcher(args)\n", "pipe": "stderr"}
{"event": "line", "data": "  File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/accelerate/commands/launch.py\", line 799, in multi_gpu_launcher\n", "pipe": "stderr"}
{"event": "line", "data": "    distrib_run.run(args)\n", "pipe": "stderr"}
{"event": "line", "data": "  File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/torch/distributed/run.py\", line 892, in run\n", "pipe": "stderr"}
{"event": "line", "data": "    elastic_launch(\n", "pipe": "stderr"}
{"event": "line", "data": "  File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/torch/distributed/launcher/api.py\", line 133, in __call__\n", "pipe": "stderr"}
{"event": "line", "data": "    return launch_agent(self._config, self._entrypoint, list(args))\n", "pipe": "stderr"}
{"event": "line", "data": "  File \"/network/scratch/o/ortizgas/data/milabench/venv/torch/lib/python3.10/site-packages/torch/distributed/launcher/api.py\", line 264, in launch_agent\n", "pipe": "stderr"}
{"event": "line", "data": "    raise ChildFailedError(\n", "pipe": "stderr"}
{"event": "line", "data": "torch.distributed.elastic.multiprocessing.errors.ChildFailedError: \n", "pipe": "stderr"}
{"event": "line", "data": "============================================================\n", "pipe": "stderr"}
{"event": "line", "data": "/home/mila/o/ortizgas/CODE/milabench/benchmarks/diffusion/main.py FAILED\n", "pipe": "stderr"}
{"event": "line", "data": "------------------------------------------------------------\n", "pipe": "stderr"}
{"event": "line", "data": "Failures:\n", "pipe": "stderr"}
{"event": "line", "data": "  <NO_OTHER_FAILURES>\n", "pipe": "stderr"}
{"event": "line", "data": "------------------------------------------------------------\n", "pipe": "stderr"}
{"event": "line", "data": "Root Cause (first observed failure):\n", "pipe": "stderr"}
{"event": "line", "data": "[0]:\n", "pipe": "stderr"}
{"event": "line", "data": "  time      : 2024-12-12_15:42:59\n", "pipe": "stderr"}
{"event": "line", "data": "  host      : cn-e002.server.mila.quebec\n", "pipe": "stderr"}
{"event": "line", "data": "  rank      : 1 (local_rank: 1)\n", "pipe": "stderr"}
{"event": "line", "data": "  exitcode  : 1 (pid: 988009)\n", "pipe": "stderr"}
{"event": "line", "data": "  error_file: <N/A>\n", "pipe": "stderr"}
{"event": "line", "data": "  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html\n", "pipe": "stderr"}
{"event": "line", "data": "============================================================\n", "pipe": "stderr"}
{"event": "end", "data": {"command": ["/home/mila/o/ortizgas/CODE/milabench/milabench/scripts/activator", "/network/scratch/o/ortizgas/data/milabench/venv/torch", "/network/scratch/o/ortizgas/data/milabench/cache", "accelerate", "launch", "--mixed_precision=bf16", "--dynamo_backend=no", "--machine_rank=0", "--num_machines=1", "--multi_gpu", "--gradient_accumulation_steps=1", "--num_cpu_threads_per_process=4", "--main_process_ip=127.0.0.1", "--main_process_port=29400", "--num_processes=4", "/home/mila/o/ortizgas/CODE/milabench/benchmarks/diffusion/main.py", "--num_epochs", "5", "--batch_size", "6", "--num_workers", "8", "--cache", "/network/scratch/o/ortizgas/data/milabench/cache"], "time": 1734036180.5733843, "return_code": 1}, "pipe": null}
